test/ts/index.test.js

```diff
@@ -6,14 +6,17 @@ const runscript = require('runscript');
 const path = require('path');
 const utils = require('../utils');
 const baseDir = path.join(__dirname, '../fixtures/apps/app-ts');
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+const rimraf = require('rimraf');
 
 describe('test/ts/index.test.js', () => {
   before(function* () {
-    if (process.env.CI) {
-      yield runscript('tsc && npmlink ../../../../', { cwd: baseDir });
-    } else {
-      yield runscript('tsc && npm link ../../../../', { cwd: baseDir });
-    }
+    yield runscript('tsc', { cwd: baseDir });
+    const dest = path.join(baseDir, 'node_modules/egg');
+    rimraf.sync(dest);
+    mkdirp.sync(path.dirname(dest));
```

Comment:
don't use npm link, it will reinstall all the node_modules

Reply:
```js
const mkdirp = require('mz-modules/mkdirp');
const rimraf = require('mz-modules/rimraf');
const fs = require('mz/fs');

yield rimraf(dest);
yield mkdirp(path.dirname(dest));
yield fs.symlink('../../../../../', dest);
```